bots/psuteam7bot/movement.js

```diff
@@ -739,25 +734,23 @@ movement.processAStarCell = (self, destination, infoMap, openQueue, closedMap) =
            nextCoordinates.y < 0) {
             continue;
         }
+        const nextCell = infoMap[nextCoordinates.y][nextCoordinates.x];
+        //If destination found, we found a path! Update this last parent and return true to indicate successful completion
+        if(movement.positionsAreEqual(nextCoordinates, destination)) {
+            nextCell.parent = current;
+            return true;
+        }
         //If coordinates not already processed, do stuff
         if(!closedMap[nextCoordinates.y][nextCoordinates.x]) {
-            const nextCell = infoMap[nextCoordinates.y][nextCoordinates.x];
-            //If destination found, we found a path! Update this last parent and return true to indicate successful completion
-            if(movement.positionsAreEqual(nextCoordinates, destination)) {
+            gNext = currCell.g + movement.getDistance(current, nextCoordinates);
+            hNext = movement.getDistance(nextCoordinates, destination);
+            fNext = gNext+hNext;
+            if(nextCell.f > fNext) {
```

Comment:
Temporary hold while we discuss best heuristic to use

Reply:
We have decided to leave the current hueristics as they seem to operate successfully. Might end up reconsidering this if we want more specific behavior related to efficient fuel use vs efficient turns taken